File: Bahasa/app/src/main/java/ds/id/Bahasa/Common/gMenuItem.java
package ds.id.Bahasa.Common;

import android.content.Context;
import android.util.AttributeSet;
import android.view.LayoutInflater;
import android.widget.TextView;
import androidx.constraintlayout.widget.ConstraintLayout;
import ds.id.Bahasa.R;

public class gMenuItem extends ConstraintLayout {

    private TextView tv1;
    private String sText;

    public gMenuItem(Context context){
        super(context);
        init(context);
    }

    public gMenuItem(Context context, AttributeSet attributeSet){
        super(context, attributeSet);
        init(context);
    }

    public gMenuItem(Context context, AttributeSet attributeSet, int defStyle) {
        super(context, attributeSet, defStyle);
        init(context);
    }

    private void init(Context context){
        LayoutInflater inflater = LayoutInflater.from(context);
        inflater.inflate(R.layout.menuitem, this, true);
        tv1 = (TextView)findViewById(R.id.tv1);
    }

    public void setSkinResource(int nResource){
        setBackgroundResource(nResource);
    }

    public void setText(String sText){
        this.sText = sText;
        tv1.setText(sText);
    }

    public String getText() {
        return sText;
    }
}

File: Bahasa(kotlin)/app/src/main/java/ds/id/Bahasa/Common/gMenuItem.kt
package ds.id.Bahasa.Common

import android.content.Context
import android.util.AttributeSet
import android.view.LayoutInflater
import android.widget.TextView
import androidx.constraintlayout.widget.ConstraintLayout
import ds.id.Bahasa.R

class gMenuItem : ConstraintLayout {

    private var tv1: TextView? = null
    private var sText: String? = null

    constructor(context: Context) : super(context) {
        init(context)
    }

    constructor(context: Context, attributeSet: AttributeSet?) : super(context, attributeSet) {
        init(context)
    }

    constructor(context: Context, attributeSet: AttributeSet?, defStyle: Int) : super(
        context,
        attributeSet,
        defStyle
    ) {
        init(context)
    }

    private fun init(context: Context) {
        val inflater = LayoutInflater.from(context)
        inflater.inflate(R.layout.menuitem, this, true)
        tv1 = findViewById<TextView>(R.id.tv1)
    }

    fun setSkinResource(nResource: Int) {
        setBackgroundResource(nResource)
    }

    fun setText(sText: String?) {
        this.sText = sText
        tv1!!.text = sText
    }

    fun getText(): String? {
        return sText
    }
}